config/ssl_config.go

```diff
@@ -22,22 +22,28 @@ import (
 )
 
 var (
+	//Deprecated: use TLSConfig instead.
```

Comment:
Nit: Please leave a space for comments, for example `//Deprecated` -> `// Deprecated`